Add copyright notices, as the next patches are going to add code.

Index: httpd-2.4.1/server/mpm/itk/itk.c
===================================================================
--- httpd-2.4.1.orig/server/mpm/itk/itk.c
+++ httpd-2.4.1/server/mpm/itk/itk.c
@@ -12,6 +12,12 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ * 
+ * Portions copyright 2005-2012 Steinar H. Gunderson <sgunderson@bigfoot.com>.
+ * Licensed under the same terms as the rest of Apache.
+ *
+ * Portions copyright 2008 Knut Auvor Grythe <knut@auvor.no>.
+ * Licensed under the same terms as the rest of Apache.
  */
 
 #include "apr.h"
Index: httpd-2.4.1/server/mpm/itk/mpm_default.h
===================================================================
--- httpd-2.4.1.orig/server/mpm/itk/mpm_default.h
+++ httpd-2.4.1/server/mpm/itk/mpm_default.h
@@ -12,6 +12,12 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ * 
+ * Portions copyright 2005-2012 Steinar H. Gunderson <sgunderson@bigfoot.com>.
+ * Licensed under the same terms as the rest of Apache.
+ *
+ * Portions copyright 2008 Knut Auvor Grythe <knut@auvor.no>.
+ * Licensed under the same terms as the rest of Apache.
  */
 
 /**
